contactql/parser.go

```diff
@@ -78,23 +79,19 @@ func (c *Condition) evaluateValue(env utils.Environment, val interface{}) (bool,
 		return stringComparison(val.(string), c.comparator, c.value)
 
 	case decimal.Decimal:
-		asDecimal, err := utils.ToDecimal(env, c.value)
+		asDecimal, err := decimal.NewFromString(c.value)
```

Comment:
this is always a regular string-> decimal conversion so shouldn't go through excellent's `ToDecimal` which doesn't things like 0/O substitutions

Reply:
Means `contactql` package only depends on `utils` and not `excellent` or `flows`